app/views/tag_groups/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<% modal ||= false %>
```

Comment:
This conditional is responsible to render the form with remote true or false.
It will be true when the form is rendered inside the modal at this line : https://github.com/Codeminer42/cm42-central/blob/bfcb5ae86fbcd0365a103a47b2ddcdce76113ba0/app/views/projects/_form.html.erb#L81